android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -806,7 +806,8 @@ private void disableElements(ReadableArray args) {
                 mBuilder = mBuilder
                         .showAnnotationsList(false)
                         .showOutlineList(false)
-                        .showUserBookmarksList(false);
+                        .showUserBookmarksList(false)
+                        .showViewLayersToolbarOption(false);
```

Comment:
why is it here?

Reply:
got it mixed up with iOS which has layers as a list